dist/wearable/js/tau.js

```diff
@@ -30404,6 +30404,20 @@ function pathToRegexp (path, keys, options) {
 				element.parentElement.insertBefore(buttonsContainer, element);
 			};
 
+			/**
+			 * remove buttons for slider
+			 * @method _removeButtons
+			 * @protected
+			 * @param {HTMLElement} element
+			 * @member ns.widget.wearable.Slider
+			 */
+			prototype._removeButtons = function (element) {
+				var buttonsContainer = document.querySelector("div .ui-slider-buttons");
```

Comment:
Hi,

Such selector (on "document") may cause problems, TAU pages are build within the same "document" and such selector may select buttons from different page.

I think that best option is to use this._ui dict to store reference to buttons container after it is build (_buildButtons method) and remove it here if it exists.